### PR TITLE
[BUG][STACK-1662][healthmonitor]: Suggest to provide more user friendly error message for non supported TLS-HELLO.

### PR DESCRIPTION
**Pull request by rahulkatre-a10**
## Description
- Severity Level: Medium
- Issue Description: TLS-HELLO healthmonitor isn’t supported, so when we try to create this kind of healthmonitor, an unhandled exception is thrown "Exception during message handling: KeyError: u'TLS-HELLO'"

## Jira Ticket
https://a10networks.atlassian.net/browse/STACK-1662

## Technical Approach
- Added exception handling code for fetching HM_type.

## Manual Testing
- Executed Health monitor create CLI cmd with "TLS-HELLO" type and observed logs.
Screenshot from 2021-02-05 20-29-37

### CODE REVIEW

File: a10_octavia/controller/worker/tasks/health_monitor_tasks.py [hunk 1]
```diff
@@ -54,11 +56,18 @@ def execute(self, listeners, health_mon, vthunder, flavor=None):
                 flavors.update(parsed_exprs)
                 args.update({'monitor': flavors})
 
+        try:
+            health_mon.type = openstack_mappings.hm_type(self.axapi_client, health_mon.type)
```
Comment (ytsai-a10) [inline a10_octavia/controller/worker/tasks/health_monitor_tasks.py:60]: Not sure is it ok to overwirte health_mon.type or not.

**Comment by NehaKembalkarA10** [comment]
To fix the travis issues occurring for unittests, the tox.ini needs to be updated by adding below requirements in it:
Add `uhashring==1.2` in `[testenv]` and `[testenv:py27]` section.
Add `cryptography<=3.3.1` in `[testenv]` section.

File: a10_octavia/controller/worker/tasks/health_monitor_tasks.py (hunk 1 above)
Comment (rahulkatre-a10) [reply a10_octavia/controller/worker/tasks/health_monitor_tasks.py:60]: We are not overwriting here, we are checking if the mapping and exception have been handled to give an appropriate error message to user.

**Comment by rahulkatre-a10** [comment]
> To fix the travis issues occurring for unittests, the tox.ini needs to be updated by adding below requirements in it:
> Add `uhashring==1.2` in `[testenv]` and `[testenv:py27]` section.
> Add `cryptography<=3.3.1` in `[testenv]` section.

Yes, I can take merge of updated master when your branch changes merged into master.

File: a10_octavia/controller/worker/tasks/health_monitor_tasks.py (hunk 1 above)
Comment (hthompson-a10) [reply a10_octavia/controller/worker/tasks/health_monitor_tasks.py:60]: +1 on the above comment. This is correct usage.

**Review by hthompson-a10 — DISMISSED** [review]

**Review by ytsai-a10 — APPROVED** [review]

**Review by hthompson-a10 — APPROVED** [review]

**Pull request merged**